components/expanded-row-alert/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+
+import { Alert, Icon } from 'design'
+
+const ExpandedRowAlert = React.memo(({ fullText, displayPage }) => {
```

Comment:
Sorry, I was not clear enough.

```jsx
<ExpandedRow>
  ...
  {fullText ? (
    <DocumentLink href={fullText.url}>Open full text</DocumentLink>
  ) : (
    <DocumentLink href={displayPage.url}>Open metadata</DocumentLink>
  )}
  ...
</ExpandedRow>
```

To simplify your life you can keep document icon for both.